Uchu.World/Objects/Components/Player/InventoryManagerComponent.cs

```diff
@@ -363,6 +363,9 @@ private static async Task<ItemComponent> GetItemComponentForLotAysnc(Lot lot)
             if (stackSize == default) 
                 stackSize = int.MaxValue;
             
+            if(settings != null)
```

Comment:
Small [style](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) comment: there should be a space between `if` and the parentheses

Reply:
The change seems small and simple, and you tested it. I don't have anything else to comment. Just that styling comment you left.